Known issue for CCO Manual mode (#26561)
openshift/openshift-docs: release_notes/ocp-4-6-release-notes.adoc

* Passthrough
* Manual

[IMPORTANT]
====

> Azure and GCP do not support the Manual mode configuration using the `install-config.yaml` file due to a known issue found in link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[BZ#1884691].

Reviewer: ```suggestion
Azure and GCP do not support the Manual mode configuration by using the `install-config.yaml` file due to a known issue found in link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[BZ#1884691].
```

routed to the provisioning network as a prerequisite.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1872787[*BZ#1872787*])

> * The {product-title} installation program does not support the Manual mode configuration using the `install-config.yaml` file on GCP and Azure due to a known issue. As a workaround, you must manually insert a ConfigMap into the manifest directory during the manifest generation stage of the cluster installation process. You can reference xref:../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[Manually creating IAM for Azure] or xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[Manually creating IAM for GCP] for more details. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[*BZ#1884691*])

Reviewer: It sounds like this configuration method is a legit option, so I suggest not calling it a workaround.

```suggestion
* The {product-title} installation program does not support the Manual mode configuration by using the `install-config.yaml` file on GCP and Azure due to a known issue. Instead, you must manually insert a ConfigMap into the manifest directory during the manifest generation stage of the cluster installation process as documented in xref:../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[Manually creating IAM for Azure] and xref:../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[Manually creating IAM for GCP]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884691[*BZ#1884691*])
```
